test/base59_test.cpp

```diff
@@ -0,0 +1,50 @@
+#include <gtest/gtest.h>
+#include <vector>
+#include <veriblock/base59.hpp>
+
+struct TestCase {
+    std::vector<uint8_t> binData;
+    std::string baseData;
+};
+
+class Base59Test : public testing::TestWithParam<TestCase> {};
+
+//  test data
+static std::vector<TestCase> g_Cases = {
+    {
```

Comment:
add some more test cases

https://github.com/VeriBlock/nodecore-js/tree/master/packages/core/testData

Here you can find two files. The one with `java` suffix, contains base59 encoded strings, that are sourced from `hex` file.

E.g. 

`base59(hex[0]) == java[0])`

Use `#include "util/literals.hpp" and `"012abcde"_unhex to get vector instead of hexstring.
E.g.
```
{"b4df3138d6b98a5a6350e653f4d6cb0c"_unhex, "GcJp9rfcTQfG104YwUGAk9"}
```

Also, use `_unhex` for current vectors - represent them as hexstring, and use `_unhex` - this makes reading much simpler

Reply:
done